app-emulation/amazon-ssm-agent/amazon-ssm-agent-2.3.1205.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+COREOS_GO_PACKAGE="${GITHUB_URI}"
+COREOS_GO_VERSION="go1.12"
+
+inherit coreos-go-depend golang-vcs-snapshot
+
+EGO_PN="github.com/aws/${PN}"
+DESCRIPTION="AWS Systems Manager Agent"
+HOMEPAGE="https://github.com/aws/amazon-ssm-agent"
+LICENSE="Apache-2.0"
+SRC_URI="https://${EGO_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz ${EGO_VENDOR_URI}"
+SLOT="0"
+KEYWORDS="amd64 arm64"
+
+S="${WORKDIR}/${PN}-${PV}/src/${EGO_PN}"
+
+PATCHES=(
+  # patch generated by `goimports -w ./agent` command
+  "${FILESDIR}"/${P}-goimports.patch
+)
+
```

Comment:
For consistency with other ebuilds, can you please use TAB for indentation, instead of 2 spaces?

Reply:
@dongsupark done. Also removed updated to fit on OEM partition